### DISCUSSION TITLE
Rendu Practical Work 1

### DISCUSSION THREAD

**Discussion by Yoy017** [heig-vd-dai-course/heig-vd-dai-course#426] [Q&A] [answered]
Bonjour,

J'ai un peu du mal à envisager de rendre le projet ce jeudi soir. Il me semble que le temps alloué à ce laboratoire a été assez limité. Je voulais donc confirmer si la date de rendu est bien fixée pour ce jeudi 17 octobre ? Si c'est le cas, je trouve que le délai est assez court. Est-ce normal ?

**Comment by ludelafo** [comment, marked as answer]
Bonjour,

Normalement, vous aviez droit à 3 semaines pour travailler sur votre travail pratique en classe (selon le planning décrit ici : <https://github.com/orgs/heig-vd-dai-course/projects/2>) :

- 27.09.2024
- 04.10.2024
- 11.10.2024

Suivi de la présentation du 18.10.2024, soit un total de 4 semaines pour effectuer le travail.

Nous pensons que le temps alloué était suffisant pour réaliser le travail et nous ne pouvons pas repousser la date de rendu pour rester équitable avec les autres classes.

Merci pour votre compréhension et je reste à votre disposition si nécessaire.

Une bonne journée,
Ludovic Delafontaine

**Reply by Yoy017** [reply]
D'accord, ça joue merci. Le temps est passé vite dans ce cas 😅

**Reply by ludelafo** [reply]
Oui, je partage votre ressenti !

Faites au mieux, concentrez-vous sur le nécessaire. Même sans un projet totalement fonctionnel, il vous sera possible d'obtenir des points. Il s'agit aussi du premier travail pratique. Vous saurez à quoi vous attendre pour les prochains travaux pratiques qui vous permettront de vous organiser au mieux.
